aetherpy/io/read_routines.py

```diff
@@ -287,6 +287,70 @@ def read_aether_ascii_header(filename):
     return header
 
 
+#  Gets dimensions, variables, attributes (may not have attributes)
+# --will require work to integrate with aetherpy
+# --not same format as GITM file headers
+# --only meant for reading block-based Aether netcdf files
+def read_blocked_netcdf_header(filename):
```

Comment:
This all belongs in a docstring for the function.  This should have:

- a one line description at the top, in imperative tense, that ends with a period
- a `Parameters` section that lists each of the input parameters and describes what they are
- a `Returns` section that lists the variables returns and describes what they are
- a `Raises` section that describes the potential errors raised and when they occur

I recommend looking at one of the other routines (e.g., line 355) for a style guide.